sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.MsTest.cs

```diff
@@ -1,11 +1,11 @@
-namespace TestMsTest
+﻿namespace TestMsTest
 {
     using System;
     using FluentAssertions;
     using Microsoft.VisualStudio.TestTools.UnitTesting;
 
     [TestClass]
-    class Program
+    public class Program
```

Comment:
I changed the example test code so that it compiles and is valid test code (apart from the deliberate mistakes such as private test method). It makes it easier to cut and paste the code into a real project to check whether it is correct.